panels/network/connection-editor/ce-page-details.c

```diff
@@ -131,6 +131,53 @@ all_user_changed (GtkToggleButton *b, CEPageDetails *page)
                 nm_setting_connection_add_permission (sc, "user", g_get_user_name (), NULL);
 }
 
+static void
+restrict_data_changed (GtkSwitch *sw, GParamSpec *pspec, CEPageDetails *page)
+{
+        NMSettingConnection *s_con;
+        NMMetered metered;
+
+        s_con = nm_connection_get_setting_connection (CE_PAGE (page)->connection);
```

Comment:
Ah, no, because `restrict_data_changed()` is only connected if `nm_connection_get_setting_connection()` returns non-`NULL` elsewhere.

⇒ Add `g_assert (s_con != NULL);` and a comment.

Reply:
Done, but, aaaargh!, forgot about the `--fixup` thing and force-pushed the branch :disappointed: